src/k8s-engine/wsl.ts

```diff
@@ -1104,6 +1104,33 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     await this.execCommand('/usr/local/bin/wsl-service', service, 'start');
   }
 
+  /**
+   * Verify that the given command runs successfully
+   * @param command
+   * @protected
+   */
+  protected async verifyReady(...command: string[]) {
+    const startTime = Date.now();
+    const maxWaitTime = 60_000;
+    const waitTime = 200;
+
+    while (true) {
+      const currentTime = Date.now();
+
+      if ((currentTime - startTime) > maxWaitTime) {
+        console.log(`Waited more than ${ maxWaitTime / 1000 } secs for ${ command.join(' ') } to succeed. Giving up.`);
+        break;
+      }
+      try {
+        await this.execCommand(...command);
+        break;
+      } catch (err) {
+        console.log(`Command ${ command } failed: ${ err }:`, err);
```

Comment:
You're including `err` twice; we probably don't need the second instance (where the stack would show).  Maybe including a message about retrying.

Do we need `console.log` here, or `console.debug`?

Reply:
Good point, should be `console.log`